enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -74,40 +74,233 @@ to implement the design.  For instance,
+| **[readiness-health-check-http-endpoint](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#readiness-health-check-http-ep)** | *string* | Endpoint called to determine if the app is ready to accept traffic.  |
+| **[readiness-health-check-invocation-timeout](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#readiness-health-check-invoc-time)** | *integer* | The timeout in seconds for individual health check requests for http and port health checks |
+| **[readiness-health-check-type](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#readiness-health-check-type)** | *string* | Type of check to perform; `none` is deprecated and an alias to `process` |
+| **instances** | *integer* | The number of instances to run |
+| **memory** | *string* | The memory limit for all instances of the web process; this attribute requires a unit of measurement: `B`, `K`, `KB`, `M`, `MB`, `G`, `GB`, `T`, or `TB` in upper case or lower case |
+| **log-rate-limit-per-second** | *string* | The log rate limit for all the instances of the process; this attribute requires a unit of measurement: `B`, `K`, `KB`, `M`, `MB`, `G`, `GB`, `T`, or `TB` in upper case or lower case, or \-1 or 0 |
+
+##### Procfiles
+
+A Procfile enables you to declare required runtime processes, called process
+types, for your app. Procfiles must be named `Procfile` exactly and placed
+in the root directory of your application.
+
+***Example***
+
+```
+web: bundle exec rackup config.ru -p $PORT
+rake: bundle exec rake
+worker: bundle exec rake workers:start
+```
+
+In a Procfile, you declare one process type per line and use the syntax
+`PROCESS_TYPE: COMMAND`.
+
+* `PROCESS_TYPE` defines the type of the process.  
+* `COMMAND` is the command line to launch the process.
+
+###### Procfile use cases
+
+Many buildpacks provide their own process types and commands by default; however,
+there are special cases where specifying a custom `COMMAND` is necessary.
+Commands can be overwritten by providing a Procfile with the same process type.
+
+For example, a buildpack may provide a `worker` process type that runs the
+`rake default:start` command. If a Procfile is provided that also contains a
+`worker` process type, but a different command such as `rake custom:start`, the
+`rake custom:start` command will be used.
+
+Some buildpacks, such as Python, that work on a variety of frameworks, do not 
+attempt to provide a default start command. For these cases, a Procfile should
+be used to specify any necessary commands for the app.
+
+###### Web process
+
+`web` is a [special process type](https://v3-apidocs.cloudfoundry.org/version/3.163.0/#web-process-type)
+that is required for all applications. The `web` `PROCESS_TYPE` must be specified
+by either the buildpack or the Procfile.
+
+###### Specifying processes in manifest files
+
+Custom process types can also be configured via a manifest file. Read more about
```

Comment:
Should we mention that we won't be supporting custom process types at this point in time?

Reply:
This is an observation for the proposal section IMO.
In CF section we're just listing the CF "state of art". I'll add a sentence in the proposal part, let me know what you think.